### ISSUE TITLE
get.helm.sh CDN 404s causing get-helm-3.sh to fail

### ISSUE THREAD

**Issue by gartmand** [helm/helm#10950]
I've been using get-helm-3 from a CodeBuild Pipeline in the us-east-1 region for the past few days. As of yesterday, it's begun consistently failing due to a checksum mismatch caused by a 404 Error. Setting DEBUG=true shows the following:

```
233 | + curl -SsL https://get.helm.sh/helm-3.8.1-linux-amd64.tar.gz.sha256 -o /tmp/helm-installer-4WALuT/helm-3.8.1-linux-amd64.tar.gz.sha256
234 | + curl -SsL https://get.helm.sh/helm-3.8.1-linux-amd64.tar.gz -o /tmp/helm-installer-4WALuT/helm-3.8.1-linux-amd64.tar.gz
235 | + verifyFile
236 | + '[' true == true ']'
237 | + verifyChecksum
238 | + printf 'Verifying checksum... '
239 | Verifying checksum... ++ awk '{print $2}'
240 | ++ openssl sha1 -sha256 /tmp/helm-installer-4WALuT/helm-3.8.1-linux-amd64.tar.gz
241 | + local sum=568ffa218e7544a2b7d6641afacb4e2c691885143fd80d834dcd14090e83e738
242 | ++ cat /tmp/helm-installer-4WALuT/helm-3.8.1-linux-amd64.tar.gz.sha256
243 | + local 'expected_sum=·<?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
244 | RequestId:81f1fb85-501e-0061-5189-646e4d000000
245 | Time:2022-05-10T16:15:37.8184779Z</Message></Error>'
246 | + '[' 568ffa218e7544a2b7d6641afacb4e2c691885143fd80d834dcd14090e83e738 '!=' '·<?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
247 | RequestId:81f1fb85-501e-0061-5189-646e4d000000
248 | Time:2022-05-10T16:15:37.8184779Z</Message></Error>' ']'
249 | + echo 'SHA sum of /tmp/helm-installer-4WALuT/helm-3.8.1-linux-amd64.tar.gz does not match. Aborting.'
250 | SHA sum of /tmp/helm-installer-4WALuT/helm-3.8.1-linux-amd64.tar.gz does not match. Aborting.
```

The 404 has been occurring almost incessantly, not only with the checksums but also with the get.helm.sh site root and the packages. As a result, I've been unable to get the build working using the script.

I'm able to reproduce this on browsers on multiple devices. Though occasionally I can get through and download something successfully, I haven't been successful at grabbing both the package and the sha256. As I'm located near the us-east-1 data centers, I can't tell if it's limited to my area. An example from my Mac:

```
Summary
---
URL: https://get.helm.sh/helm-3.8.1-linux-amd64.tar.gz.sha256
Status: 404
Source: Network

Request
---
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Safari/605.1.15

Response
---
Date: Tue, 10 May 2022 16:40:52 GMT
Content-Length: 215
Content-Type: application/xml
Server: Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0
x-ms-version: 2009-09-19
x-ms-request-id: 93ddc4e3-101e-0094-0a8c-64fa67000000
```

**Comment by gartmand** [comment]
A member of my team who is located in another region has confirmed that https://get.helm.sh/ returns a 404
```
<Error>
<Code>ResourceNotFound</Code>
<Message>The specified resource does not exist. RequestId:5d8d5a05-c01e-002e-1e91-641f19000000 Time:2022-05-10T17:13:10.7157020Z</Message>
</Error>
```

**Comment by jdolitsky** [comment]
@mattfarina - possibly related to 3.9.1 rc? https://github.com/helm/helm/releases/tag/v3.9.0-rc.1

**Comment by gartmand** [comment]
@jdolitsky I'm going to hazard a guess that it has something to do with the version string that I messed up. There's a chance they were aliased before (with / without the v) and aren't now, but in likelihood when I moved my pin I forgot to include the v. In any case, this is probably user error. I'm going to close and if it pops up again, I'll reopen.

(There's a chance get.helm.sh root never returned anything but 404, I didn't try to hit it before)
